pkg/kubelet/cm/dra/state/checkpoint.go

```diff
@@ -34,9 +41,44 @@ type DRAManagerCheckpoint struct {
 	Checksum checksum.Checksum  `json:"checksum"`
 }
 
+// DraManagerCheckpoint struct is an old implementation of the DraManagerCheckpoint
+type DRAManagerCheckpointWithoutResourceHandles struct {
+	Version  string                                   `json:"version"`
+	Entries  ClaimInfoStateListWithoutResourceHandles `json:"entries,omitempty"`
+	Checksum checksum.Checksum                        `json:"checksum"`
+}
+
 // List of claim info to store in checkpoint
 type ClaimInfoStateList []ClaimInfoState
 
+// ClaimInfoStateWithoutResourceHandles is an old implementation of the ClaimInfoState
+type ClaimInfoStateWithoutResourceHandles struct {
+	// Name of the DRA driver
+	DriverName string
+
+	// ClassName is a resource class of the claim
+	ClassName string
+
+	// ClaimUID is an UID of the resource claim
+	ClaimUID types.UID
+
+	// ClaimName is a name of the resource claim
+	ClaimName string
+
+	// Namespace is a claim namespace
+	Namespace string
+
+	// PodUIDs is a set of pod UIDs that reference a resource
+	PodUIDs sets.Set[string]
+
+	// CDIDevices is a map of DriverName --> CDI devices returned by the
+	// GRPC API call NodePrepareResource
+	CDIDevices map[string][]string
+}
```

Comment:
This should go next to the declaration of the "normal" ClaimInfo state (either move that one here or this one there).

Reply:
Thank you for the reveiw! done.